.github/workflows/current.yml

```diff
@@ -0,0 +1,53 @@
+name: Test on current Node.js
+
+on:
+  schedule:
+    - cron: '00 00 * * 1'
+  workflow_dispatch:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    services:
```

Comment:
This isn't specific to this PR, but I notice a lot of boilerplate in our workflow `.yml` files. Is there a such thing as includes or some other way to de-duplicate?

Reply:
It looks like their current options to [avoid duplication](https://docs.github.com/en/actions/sharing-automations/avoiding-duplication) are intended for a much larger scale. At a glance, setting up composite actions might help - bundling together checkout, setup-node, and `npm ci` - but it's not obvious to me that the added complexity is a better trade-of than the duplication.